runtime/util/vmargs.c

```diff
@@ -1512,6 +1547,11 @@ addLauncherArgs(J9PortLibrary * portLib, JavaVMInitArgs *launcherArgs, UDATA lau
 	return 0;
 }
 
+#undef JAVA_CLASS_PATH_EQUALS
```

Comment:
why are these undef'd

Reply:
Since we don't use these macros anywhere else in the code, may as well undef them here no? I can remove this if it doesn't make sense

Reply:
> Since we don't use these macros anywhere else in the code

This is true, but we typically don't do this. There are a lot of macros in the codebase that we only use once.